cstrike/addons/amxmodx/scripting/csgo_clans.sma

```diff
@@ -2214,9 +2214,9 @@ public say_text(msgId, msgDest, msgEnt)
 	        set_msg_arg_string(4, "");
 
 	        add(message, charsmax(message), prefix);
-	        add(message, charsmax(message), "^3 ");
+	        add(message, charsmax(message), "^3");
```

Comment:
That whitespace is actually required to properly display a message.
With those changes, it would look like this: `[CLAN]PlayerName: Message` instead of: `[CLAN] PlayerName : Message`.

Reply:
I think the space isn't needed before the ':' symbol, but after the clan name you are right.
Going to edit it.

Reply:
The default format is: `PlayerName : Message`, so for consistency, it should be the same.

Reply:
Done.